routes/web.php

```diff
@@ -23,3 +23,7 @@
 });
 
 Route::get('/auth/logout', [AuthController::class, 'logout'])->name('auth.logout');
+
+Route::middleware('auth')->namespace('App\Http\Livewire')->group(function () {
+    Route::get('/airfields', 'Airfields\Index');
+});
```

Comment:
We should decide on a format here. For example, the logout route above uses a different syntax to this. There isn't a right one, although the logout approach does allow for IDE's to CMD+Click through to the class more easily.

Reply:
The problem you'll get if you have 5 Index livewire components, you'll spend more time aliasing. Happy to do whatever the consensus is.

Reply:
I don't think there is anything wrong with avoiding imports and just using \App\Livewire\Components\Airfields\Index::class so it is really clear

Reply:
Changed